easybuild/easyblocks/p/python.py

```diff
@@ -82,8 +82,21 @@
     if debug:
         print("[%(EBPYTHONPREFIXES)s] postfix subdirectory to consider in installation directories: %%s" %% postfix)
 
-    site_packages = site.getsitepackages()
-    base_paths = [p for p in sys.path if p in site_packages]
+    try:
+        site_packages = site.getsitepackages()
+        base_paths = [p for p in sys.path if p in site_packages]
+    except AttributeError:
+        # Workaround for old(?) virtualenvs
```

Comment:
please include a reference to https://github.com/pypa/virtualenv/issues/737 in a comment here

Reply:
Done